### PR TITLE
Add module offset

### PR DESCRIPTION
**Pull request by Jupeyy**
Get module offset, so we can at least recover the DDNet.exe paths:
`Error occurred on Wednesday, February 23, 2022 at 20:34:57. (with offset 00007FF7C4A00000)`
(Address - Offset) + (Image-Base)
(ImageBase  = winedump -f DDNet.exe (image base: usually 0x140000000)

Tested on windows 64-bit

https://github.com/Jupeyy/drmingw/commit/08ab91c4897c04b5919d14fc2d7c215015d0fb94

ez download from github actions:
https://github.com/Jupeyy/drmingw/actions/runs/1889238377

## Checklist

- [ ] Tested the change ingame
- [ ] Provided screenshots if it is a visual change
- [ ] Tested in combination with possibly related configuration options
- [ ] Written a unit test if it works standalone, system.c especially
- [ ] Considered possible null pointers and out of bounds array indexing
- [ ] Changed no physics that affect existing maps
- [ ] Tested the change with [ASan+UBSan or valgrind's memcheck](https://github.com/ddnet/ddnet/#using-addresssanitizer--undefinedbehavioursanitizer-or-valgrinds-memcheck) (optional)

### CODE REVIEW

File: src/base/system.cpp [hunk 1]
```diff
@@ -4072,8 +4072,17 @@ void init_exception_handler()
 	gs_ExceptionHandlingModule = LoadLibraryA("exchndl.dll");
 	if(gs_ExceptionHandlingModule != nullptr)
 	{
-		auto pfnExcHndlInit = (void APIENTRY (*)())GetProcAddress(gs_ExceptionHandlingModule, "ExcHndlInit");
-		pfnExcHndlInit();
+		// Intentional
```
Comment (def-) [inline src/base/system.cpp:4075]: Why does this change contain my parts again? Maybe rebase

Comment (def-) [reply src/base/system.cpp:4075]: Oh wait, this is in another location, nevermind

**Comment by def-** [comment]
Fine like this? https://github.com/ddnet/ddnet-libs/pull/26

**Comment by Jupeyy** [comment]
I'll test 32 bit tomorrow to be safe, but generally yes

**Comment by Jupeyy** [comment]
I added a script that automatically parses the crash log, so it's easier to obtain the symbols

**Comment by Jupeyy** [comment]
I also tested 32bit as said

**Comment by Jupeyy** [comment]
```
TMP_OFFSET=$(cat "$2" | grep -E -o "\(with offset [0-[9](https://github.com/ddnet/ddnet/runs/5316844302?check_suite_focus=true#step:9:9)A-F]*\)" | grep -E -o "[A-F0-9]*")
                 ^--^ SC2002: Useless cat. Consider 'cmd < file | ..' or 'cmd file | ..' instead.
```

I dunno what's shellchecks problem, but if somebody can fix it pls do

**Comment by def-** [comment]
Done

**Comment by Jupeyy** [comment]
> Done


Thanks

**Pull request merged**